opm/autodiff/SimulatorFullyImplicitBlackoilOutput.hpp

```diff
@@ -428,9 +435,9 @@ namespace Opm
 
         const Wells* wells = wellsmanager.c_wells();
         wellstate.resize(wells, simulatorstate); //Resize for restart step
-        auto restarted = Opm::init_from_restart_file(
-                                eclipseState_,
-                                Opm::UgGridHelpers::numCells(grid) );
+        auto restarted = Opm::load_from_restart_file(eclipseState_,
```

Comment:
No - that is right; it was there when this PR was originally crafted, then I ended up updating https://github.com/OPM/opm-output/pull/152; updates to this PR will come as well.

So yes - right now the method:

```C++
Opm::EclipseIO::load_from_restart_file()
```
is correct. Might change again though.